compression/cutils/tests/tests.ts

```diff
@@ -0,0 +1,77 @@
+import * as anchor from "@coral-xyz/anchor";
+import {decode, getAccounts, mapProof} from "./utils/utils";
+import {SPL_ACCOUNT_COMPRESSION_PROGRAM_ID, SPL_NOOP_PROGRAM_ID} from "@solana/spl-account-compression";
+import {getAsset, getAssetProof} from "./utils/readAPI";
+import {Cutils} from "../target/types/cutils";
+import {loadOrGenerateKeypair, loadPublicKeysFromFile} from "./utils/helpers";
+import { PROGRAM_ID as BUBBLEGUM_PROGRAM_ID } from "@metaplex-foundation/mpl-bubblegum/dist/src/generated";
+
+describe("cutils", () => {
+    const provider = anchor.AnchorProvider.env();
+    anchor.setProvider(provider);
+
+    const program = anchor.workspace.Cutils as anchor.Program<Cutils>;
+
+    // NFT metadata pointer
+    // TODO change
+    const uri = "https://arweave.net/nVRvZDaOk5YAdr4ZBEeMjOVhynuv8P3vywvuN5sYSPo"
+
+    const payer = loadOrGenerateKeypair("payer");
+
+    // cNFT receiver
+    const testWallet = loadOrGenerateKeypair("testWallet")
+
+    const {collectionMint, treeAddress} = loadPublicKeysFromFile()
+
+    it("Mint!", async () => {
+        const tx = await program.methods.mint({uri})
+            .accounts({
+                payer: payer.publicKey,
+                leafOwner: testWallet.publicKey,
+                leafDelegate: testWallet.publicKey, //verify
+                treeDelegate: payer.publicKey,
+                collectionAuthority: payer.publicKey,
+                collectionAuthorityRecordPda: BUBBLEGUM_PROGRAM_ID,
+                ...getAccounts(collectionMint, treeAddress)
+            })
+            .transaction()
+
+        const sx = await program.provider.sendAndConfirm(tx, [payer], {skipPreflight: true});
+        console.log(`   Tx Signature: ${sx}`);
+    });
+
+
+    // it("Verify", async () => {
```

Comment:
Is the Verify not working yet?

Reply:
It is working, just that it might be best to execute individually, and not as a whole for the entire test suite

So I settled for uncommenting and commenting code lol

I'll just create throw an error in the verify test if the assetID is empty